### PR TITLE
Theme sphinx-book-theme

### PR DESCRIPTION
**Pull request by ksuess**
See requirements: https://github.com/plone/training/issues/506

DONE
- [x] copy code with click on 'copy' button
- [x] sticky toc
- [x] main navigation sticky (fixed in https://github.com/plone/training/pull/507/commits/b05fe223f9722384aad4aed43fbc4ea7fd61b496)
- [x] exercises with collapsed solution

TODO
- [ ] sidebar

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,5 +4,8 @@ Sphinx
 sphinx-rtd-theme
 sphinxcontrib-spelling
 sphinxcontrib-websupport
+sphinx-copybutton
+sphinx-togglebutton
+sphinx-panels
```
Comment (stevepiercy) [inline requirements.txt:9]: I found that `sphinx-panels` is not compatible with this theme.

https://github.com/executablebooks/sphinx-panels/issues/57

We can upvote it to get more attention to the issue.

`sphinx-togglebutton` is fine for what we need.

**Comment by spereverde** [comment]
- I added a poc setup for myst inside the the `relaunch-book-theme` branch with the folder for *transmogrifier* docu migrated from `rst` to `myst` markdown
Would you like me to push this to this branch or create a new branch based on this one or ...?

- I have some small docu on installation and conversion for this, where should I add it, in this PR, the #506  issue, plone_training_config folder, ...

**Comment by ksuess** [comment]
I'ts great to see the markdown idiom **MyST** here in the training. I do understand now that it needs a conversation, not a side by side use of both reStructuredText and a markdown idiom. But a decision which one to take would make it clear and easy to know for the future authors what language to use for authoring. And BTW in fact I personally think it' easy to learn both.

I like your approach to start with one single training to see how the conversation of existing material works. If it's not a too big effort, would you take as an example the training 'Mastering Plone Development' as there are some additional features like an extra sidebar for code checkouts and there are exercises with solutions.

For your docu how to install and apply the conversation, I think it would be good to have it here in the repo. I propose to create a '_doc' folder with your documentation about myst and maybe more in the future. What do you think?

**Comment by spereverde** [comment]
Thanks for the feedback!
I will add the documentation in a _doc folder, sounds good to me also.
Of course I started with an old training, because that seemed simpler :-) 
We'll see what the magic does for something huge like the *Mastering Plone Development*
I'll make a copy so you can see still see them side by side.

**Comment by ksuess** [comment]
@spereverde awsome!

**Comment by stevepiercy** [comment]
@spereverde @ksuess I would prefer not to combine two purposes into one PR, please. It is much easier for me to review focused and smaller changes.

This PR and its branch `relaunch-book-theme` seems to be a proof of concept (POC) that sphinx-book-theme preserves the features of the previous `sphinx_rtd_theme` without significantly changing content or markup except as necessary.

Another PR could be a POC for, and documentation of, converting reStructuredText markup to MyST markup. We could convert content from reST to MyST after fiddling with the new theme.

**Comment by spereverde** [comment]
PR https://github.com/plone/training/pull/508 for myst added

**Comment by ksuess** [comment]
Replaced by future PR of branch https://github.com/plone/training/tree/ksuess-theme-playground

**Pull request closed without merging**